policyengine_us/parameters/gov/states/wv/tax/income/credits/heptc/low_income_guidelines.yaml

```diff
@@ -0,0 +1,20 @@
+description: West Virginia qualifies filers for the homestead excess property tax credit with income below this low income poverty guidelines.  
+first_person:
+  2012-01-01: 40_770
+
+additional_person:
+  2012-01-01: 14_160
+
+metadata:
+  unit: currency-USD
+  reference:
+  - title: West Virginia Personal Income Tax Forms And Instructions 2022
+    href: https://tax.wv.gov/Documents/PIT/2022/PersonalIncomeTaxFormsAndInstructions.2022.pdf#page=14
+  - title: 2021 West Virginia Personal Income Tax Forms And Instructions-2021 FAMILY TAX CREDIT TABLES
+    href: https://tax.wv.gov/Documents/TaxForms/2021/it140.pdf#page=13 
+  - title: Code of West Virginia §11-21-23 (h)(3)
```

Comment:
These numbers aren't here. Can you find where the legal code defines it?

Reply:
![image](https://github.com/PolicyEngine/policyengine-us/assets/104268340/542b93e2-c388-4bff-bf1b-5660e1528192)
Do we have this parameter on the federal level?

Reply:
Yes `fpg.yaml` but you can use the `tax_unit_fpg` variable directly

Reply:
40,000 seems a lot higher than our fpg.yaml , not sure why that is

Reply:
300%

Reply:
So do I need to set 300% as a parameter and directly use 300% x tax_unit_fpg instead of using the accurate numbers?

Reply:
Do the numbers differ from 300% of fpl?

Reply:
the numbers are exactly 300% of fpg